packages/core/src/render3/bindings.ts

```diff
@@ -59,8 +59,11 @@ export function bindingUpdated(lView: LView, bindingIndex: number, value: any):
       if (!devModeEqual(oldValueToCompare, value)) {
         const details =
             getExpressionChangedErrorDetails(lView, bindingIndex, oldValueToCompare, value);
+
+
         throwErrorIfNoChangesMode(
-            oldValue === NO_CHANGE, details.oldValue, details.newValue, details.propName);
+            oldValue === NO_CHANGE, details.oldValue, details.newValue, details.propName,
+            details.componentClassName);
```

Comment:
I'd propose passing an LView here directly and get the necessary info (component class name) inside of the function:
```suggestion
            lView);
```